src/Analysis/GWASResults/Components/Diagrams/lib/pheweb_plots.js

```diff
@@ -40,6 +40,7 @@ import {memoize, property, range, some, sortBy, template} from 'lodash';
 /* eslint prefer-destructuring: 0 */ // --> OFF
 /* eslint wrap-iife: 0 */ // --> OFF
 /* eslint arrow-parens: 0 */ // --> OFF
+/* eslint no-inner-declarations: 0 */ // --> OFF
 
```

Comment:
Adding this would remove ~50 warnings from the console: 
```/* eslint func-names: 0 */ // --> OFF```

or just disabling eslint for the file, since we're not planning on maintaining their library code.

Reply:
done